rules/naming/tests/ValueObjectFactory/PropertyRenameFactory/PropertyRenameFactoryTest.php

```diff
@@ -33,10 +32,8 @@ final class PropertyRenameFactoryTest extends AbstractKernelTestCase
      */
     private $betterNodeFinder;
 
-    public function __construct(?string $name = null, array $data = [], $dataName = '')
+    protected function setUp(): void
```

Comment:
@dobryy I guess why you used `__construct`... 
Yet `__construct` should never by used in TestCase
PHPStan rule for this is waiting - https://github.com/symplify/symplify/issues/2225

Follow bellow...